.github/workflows/preview.yml

```diff
@@ -46,6 +46,12 @@ jobs:
       shell: bash
       run: echo "branch=$(echo ${GITHUB_REF#refs/heads/})" >> $GITHUB_OUTPUT
       id: extract_branch
+    # - name: link to checklist
+    #   if: ${{ github.event_name == 'pull_request' && github.event.action == 'opened' && startsWith(github.event.pull_request.title, 'ingestion') == true}}
+    #   uses: trixi-framework/add-pr-review-checklist@v1
+    #   with:
+    #     file: '.github/ingestion-review-checklist.md'
+    #     no-checklist-keyword: '[no checklist]'
     - name: list volumes
```

Comment:
This can be removed, right? Is there any reason to keep it in?

Reply:
Yes, this can be removed.